triage/2022-03-22.md

```diff
@@ -0,0 +1,27 @@
+# 2022-03-22 Triage Log
+
+Very quiet week for performance, with just one statistically significant change
+landing in the last week. This change was a regression, though primarily in
+stress tests, and was a result of a soundness fix.
+
+Triage done by **@simulacrum**.
+Revision range: [3ba1ebea122238d1a5c613deb1bf60ce24bd8fd8..3ea44938e21f0de8ae7d4f6399a8a30f97867c70](https://perf.rust-lang.org/?start=3ba1ebea122238d1a5c613deb1bf60ce24bd8fd8&end=3ea44938e21f0de8ae7d4f6399a8a30f97867c70&absolute=false&stat=instructions%3Au)
+
+1 Regressions, 0 Improvements, 0 Mixed; 0 of them in rollups
+37 comparisons made in total
+
+#### Regressions
+
+Disable almost certainly unsound early otherwise branch MIR opt [#95161](https://github.com/rust-lang/rust/pull/95161)
+- Arithmetic mean of relevant regressions: 3.1%
+- Largest regression in [instruction counts](https://perf.rust-lang.org/compare.html?start=3c17c84a386e7badf6b2c6018d172496b3a28a04&end=cf8531064e44a335c7da75c745279457f42660c5&stat=instructions:u): 8.5% on `full` builds of `deeply-nested-multi check`
+- Minimal impact on primary benchmarks, and as a soundness fix naturally pretty
+  justified.
+
+#### Improvements
+
+- No improvements.
+
+#### Mixed
+
+- No mixed results.
```

Comment:
Are we no longer including the untriaged PRs?

Reply:
I am dropping them, yeah, the list in the generation is just some subset of 30 but there's >100 now IIRC and I don't think anyone is actively investing in that triage.

Reply:
Ok at some point we should talk about how we want to handle regressions going forward.